Mlem/Views/Shared/TimestampView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  TimestampView.swift
+//  Mlem
+//
+//  Created by Sjmarf on 07/07/2023.
+//
+
+import SwiftUI
+
+struct TimestampView: View {
+    
+    let date: Date
+    var spacing: CGFloat = 4
+    
+    var body: some View {
+        HStack(spacing: spacing) {
```

Comment:
Spacing should use `AppConstants.iconToTextSpacing`

Reply:
There is no such `AppConstants.iconToTextSpacing` on my branch. My implementation of `TimestampView` accepts a `spacing` argument - by passing a value as this argument, all instances of `TimestampView` are able to use the same spacing as they did before this PR. 

The values `PostInteractionBar.iconToTextSpacing` and `CommentInteractionBar.iconToTextSpacing` are passed as the `spacing` argument in the relevant `TimestampView` instances. Is that what you meant?

Reply:
It's on master, must have come in a recent PR. I'll approve, once the merge conflicts are fixed we're good to go

Reply:
Ah I see. I've changed it 👍